From 60b9944c170f726d57126058ceae2a65aac3693b Mon Sep 17 00:00:00 2001
From: "richard.feliciano" <richard.feliciano@logicpd.com>
Date: Mon, 23 Dec 2019 15:37:07 -0600
Subject: [PATCH] Added changes so Laird wlan/bluetooth components can be built
 with the LogicPD iMX8MM kit


diff --git a/recipes-kernel/lwb-radio-firmware/lwb5-radio-firmware-fcc_7.0.0.142.bbappend b/recipes-kernel/lwb-radio-firmware/lwb5-radio-firmware-fcc_7.0.0.142.bbappend
new file mode 100644
index 0000000..41949c0
--- /dev/null
+++ b/recipes-kernel/lwb-radio-firmware/lwb5-radio-firmware-fcc_7.0.0.142.bbappend
@@ -0,0 +1,4 @@
+do_install() {
+	install -d  ${D}${nonarch_base_libdir}
+	tar -xjf ${S}/${LWB_PART_NAME}.tar.bz2 -C ${D}${nonarch_base_libdir} --strip-components=1 --no-same-owner
+}
-- 
2.17.1

